src/commands/shane/permset/create.ts

```diff
@@ -98,28 +98,31 @@ export default class PermSetCreate extends SfdxCommand {
             label: this.flags.name
         });
 
-        const objectList: Set<string> = new Set<string>();
+        let objectList: Array<string> = new Array<string>();
```

Comment:
Actually, this whole file doesn't make sense.  Is your TS version not supporting Set?  
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

It's legit.

Reply:
It does, but then I'm getting an error later on for (.. of ..) because a Set isn't iterable. It actually works though, but I wasn't able to get rid of this TS error: Type 'Set<string>' is not an array type or a string type. Use compiler option '--downlevelIteration' to allow iterating of iterators.

Reply:
Plus the line `this.ux.log(Object list is ${objectList});` wasn't working correctly. Not sure exactly, but it was displaying something like 'Object list is [Set]'.